backend/package.json

```diff
@@ -24,9 +24,9 @@
     "express": "^4.18.2",
     "express-jwt": "^8.4.1",
     "jsonwebtoken": "^9.0.0",
+    "nodemon": "^2.0.20",
```

Comment:
added nodemon so that it keeps watching all the changes in our files in the backend - now we dont need to do npm start every time we make a change in server.js